### PR TITLE
fix(ldap): Fix SQL injection on LDAP page

### PR DESCRIPTION
**Pull request by callapa**
## Description

When a user is connected he can perform an SQL injection (Time-Based SQL Injection Attacks) on the following page:
`.../centreon/include/Administration/parameters/ldap/xml/ldap_host.php`

**Fixes** # (CVE-2019-15300)

After verification, this file is no longer used in Centreon.
We have deleted this file and all the others related to it.

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [x] 19.10.x (master)

<h2> How this pull request can be tested ? </h2>

please contact us

#### Community contributors & Centreon team

- [x] I followed the **coding style guidelines** provided by Centreon
- [x] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [x] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

#### Centreon team only

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests cover 80%** of the code written for the story.
- [x] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

File: www/install/sql/centreon/Update-DB-19.10.2.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+UPDATE topology SET topology_url = NULL, topology_url_opt = NULL WHERE topology_page = 501;
```
Comment (sc979) [inline www/install/sql/centreon/Update-DB-19.10.2.sql]: shouldn't this should be added in the last update file ?

File: www/include/Administration/parameters/ldap/form.php [hunk 2]
```diff
@@ -1,34 +1,18 @@
...
- * GPL Licence 2.0.
- *
- * This program is free software; you can redistribute it and/or modify it under
- * the terms of the GNU General Public License as published by the Free Software
- * Foundation ; either version 2 of the License.
- *
- * This program is distributed in the hope that it will be useful, but WITHOUT ANY
- * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
- * PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ * Copyright 2005 - 2019 Centreon (https://www.centreon.com/)
```
Comment (sc979) [inline www/include/Administration/parameters/ldap/form.php]: please restore GPL license

File: www/include/Administration/parameters/ldap/javascript/ldapJs.php [hunk 3]
```diff
@@ -1,92 +1,35 @@
 <?php
 /*
- * Copyright 2005-2019 Centreon
- * Centreon is developed by : Julien Mathis and Romain Le Merlus under
- * GPL Licence 2.0.
+ * Copyright 2005 - 2019 Centreon (https://www.centreon.com/)
```
Comment (sc979) [inline www/include/Administration/parameters/ldap/javascript/ldapJs.php]: please restore GPL license

File: www/install/sql/centreon/Update-DB-19.10.2.sql (hunk 1 above)
Comment (callapa) [reply www/install/sql/centreon/Update-DB-19.10.2.sql]: File deleted

**Pull request merged**